modules/oe_content_persistent/src/Form/PurlSettingsForm.php

```diff
@@ -50,9 +91,27 @@ public function buildForm(array $form, FormStateInterface $form_state) {
   public function submitForm(array &$form, FormStateInterface $form_state) {
     $this->configFactory->getEditable('oe_content_persistent.settings')
       ->set('base_url', $form_state->getValue('base_url'))
+      ->set('supported_entity_types', $form_state->getValue('supported_entity_types'))
       ->save();
 
     parent::submitForm($form, $form_state);
   }
 
+  /**
+   * Returns an array of available entity types in the site.
+   *
+   * @return array
+   *   The available entity types keyed by ID.
+   */
+  protected function getEntityTypeOptions() : array {
+    $entity_type_definitions = $this->entityTypeManager->getDefinitions();
+    $entity_type_options = [];
+    foreach ($entity_type_definitions as $definition) {
+      if ($definition->entityClassImplements('Drupal\Core\Entity\ContentEntityInterface')) {
```

Comment:
Arent the definitions ContentEntityType and ConfigEntityType? Just check instanceof

<img width="697" alt="Screenshot 2020-07-31 11 59 13" src="https://user-images.githubusercontent.com/5848933/89024378-4bf7c080-d325-11ea-86e1-b1714e248456.png">